Tasks/IISWebAppManagementOnMachineGroupV0/Utility.ps1

```diff
@@ -212,7 +212,15 @@ function Get-CustomCredentials {
         [string] $password 
     )
 
-    $secretPassword = "$password" | ConvertTo-SecureString -AsPlainText -Force
+    if ($password -eq "")
+    {
+        $secretPassword = New-Object System.Security.SecureString
```

Comment:
Do we need to fix this in WinRm task as well ?

Reply:
Yes. Will raise a PR for that.